samples/IdentitySample.Mvc/Properties/launchSettings.json

```diff
@@ -16,7 +16,9 @@
       }
     },
     "web": {
-      "commandName": "web",
+      "commandName": "Project",
```

Comment:
`Project` is  for running application in the console host. To see the log messages on the console.

`web` is the name you click to switch between `IIS Express` and `console host`.
![web](https://cloud.githubusercontent.com/assets/1936364/22288693/b2e5706c-e300-11e6-9977-3a228a1e0dd0.PNG)